tests/FsCodec.NewtonsoftJson.Tests/VerbatimUtf8ConverterTests.fs

```diff
@@ -80,7 +96,12 @@ module VerbatimUtf8Tests = // not a module or CI will fail for net461
         let des = JsonConvert.DeserializeObject<Batch>(ser, defaultSettings)
         let loaded = FsCodec.Core.TimelineEvent.Create(-1L, des.e.[0].c, des.e.[0].d)
         let decoded = defaultEventCodec.TryDecode loaded |> Option.get
-        x =! decoded
+        match x, decoded with
+        | U.Double x, U.Double d when Double.IsNaN x && Double.IsNaN d -> ()
```

Comment:
You can preserve the symmetry of having a type check either way by doing something like
`| U.Double x, U.Double d when Double.IsNaN x => test <@ Double.IsNaN d @>` (or `true =! Double.isNan d` is you prefer)